lib/style/module_directives.ex

```diff
@@ -86,110 +130,62 @@ defmodule Styler.Style.ModuleDirectives do
         {directive, _, _} -> directive
       end)
 
-    # TODO: (optimization)
-    # now that we have use/import/alias/require, we might as well run
-    # them through the sort/expand/dedupe functionality and skip them in the traversal
     shortdocs = directives[:"@shortdoc"] || []
-    moduledocs = directives[:"@moduledoc"] || if needs_moduledoc?(name), do: [@moduledoc_false], else: []
-    # TODO sort behaviours?
-    behaviours = directives[:"@behaviour"] || []
-    behaviours = List.update_at(behaviours, -1, &set_newlines(&1, 2))
+    moduledocs = directives[:"@moduledoc"] || if add_moduledoc?, do: [@moduledoc_false], else: []
+    behaviours = expand_and_sort(directives[:"@behaviour"] || [])
 
-    uses = directives[:use] || []
-    imports = directives[:import] || []
-    aliases = directives[:alias] || []
-    requires = directives[:require] || []
+    uses = (directives[:use] || []) |> Enum.flat_map(&expand_directive/1) |> reset_newlines()
 
-    children =
+    imports = expand_and_sort(directives[:import] || [])
+    aliases = expand_and_sort(directives[:alias] || [])
+    requires = expand_and_sort(directives[:require] || [])
+
+    directives =
       Enum.concat([
         shortdocs,
         moduledocs,
         behaviours,
         uses,
         imports,
         aliases,
-        requires,
-        other
```

Comment:
I'm probably just blind, but... where was `other` coming from before? 🙃

Reply:
`{directives, other} = Enum.split(...)`

Reply:
other is now just called `nondirectives`